### ISSUE TITLE
bug: chapter number incorrect

### ISSUE THREAD

**Issue by jaroslaw-weber** [ProfXwing/copytype#6]
<img width="1435" alt="Screenshot 2023-10-02 at 12 11 11 AM" src="https://github.com/ProfXwing/copytype/assets/9774233/159eb168-a131-4580-93ae-3f50244b91ad">

it's still first chapter but showing as 6th.

maybe it would be better to display as percentage of the book read, would be probably easier to implement and and more precise.
maybe some progress bar on bottom on the screen with some text like "progress: 3%" or "page 4/200"

**Comment by ProfXwing** [comment]
agreed, progress bar would be more precise. I found it more difficult to implement, somehow, but I think it would be easy now
